components/style/properties/properties.mako.rs

```diff
@@ -3085,9 +3090,338 @@ pub fn cascade(
         font_metrics_provider,
         flags,
         quirks_mode,
+        registered_property_set.read_with(guards.registered_property_set),
```

Comment:
I'd require just a `&RegisteredPropertySet`, and move the `read_with` to the caller when possible here, but no big deal.

Reply:
Did for consistency with other arguments to `cascade`.